osu.Game/Screens/Menu/SupporterDisplay.cs

```diff
@@ -111,7 +112,7 @@ protected override void LoadComplete()
 
             this
                 .FadeOut()
-                .Delay(1000)
+                .Delay(RNG.Next(800, 4000))
```

Comment:
...random delay? is this intentional? why?

Reply:
I just like having it not always show at the same point. Break up the routine a bit.